packages/kbn-analytics/src/report.ts

```diff
@@ -20,7 +24,7 @@ export interface Report {
       key: string;
       appName: string;
       eventName: string;
-      type: UiCounterMetricType;
+      type: string;
```

Comment:
nit: How about allowing both? Would that help with code autocompletion when using any of the known `UiCounterMetricType`?
```suggestion
      type: UiCounterMetricType | string;
```

Reply:
Oh! I just noticed that `UiCounterMetricType` already allows `string`. Any reason for this change here then?

Reply:
I think we can keep UiCounterMetricType as a type, just to keep it consistent across all components using the same type.